doc/sphinx/credits.rst

```diff
@@ -1199,25 +1199,25 @@ Credits: version 8.7
 and cleanups of the internals of the system along with a few new features. The
 main user visible changes are:
 
-- New tactics: variants of tactics supporting existential variables eassert,
-  eenough, etc... by Hugo Herbelin. Tactics extensionality in H and
-  inversion_sigma by Jason Gross, specialize with ... accepting partial bindings
+- New tactics: variants of tactics supporting existential variables :tacv:`eassert`,
+  :tacv:`eenough`, etc... by Hugo Herbelin. Tactics ``extensionality in H`` and
+  :tacv:`inversion_sigma` by Jason Gross, ``specialize with ...`` accepting partial bindings
   by Pierre Courtieu.
 
-- Cumulative Polymorphic Inductive Types, allowing cumulativity of universes to
+- ``Cumulative Polymorphic Inductive Types``, allowing cumulativity of universes to
```

Comment:
I don't see why this should be displayed as teletyped.

Reply:
As far as I am concerned, the uppercased letters made it an obvious reference to the vernacular command `Cumulative Polymorphic Inductive`. (Putting Types inside the teletype part was an error of mine.)

Reply:
Indeed, and and types should probably have been lowercase.